gadgetron/meta.yaml

```diff
@@ -11,25 +11,28 @@ requirements:
     - armadillo>=9.900.5
     - boost=1.76.0
     - cmake>=3.20.0
-    - cuda-libraries-dev
-    - cuda-libraries
-    - cuda-runtime
-    - cuda-nvcc
+    - cuda-libraries-dev      # [linux64]
```

Comment:
Does this mean that we are not building with CUDA support on the Mac?

Reply:
For now, yes.

When I just checked on anaconda.org, there was a real dearth of macOS support on [Nvidia's organization page](https://anaconda.org/nvidia).  Even [Anaconda's CUDA packages](https://anaconda.org/anaconda/cudatoolkit) for macOS are woefully behind the other platforms.

I don't have access to a Mac with a real GPU, so I have no way to test.  The [GitHub hosted runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources) don't seem to have GPUs either.  I know that doesn't stop the builds on Linux, but till we have a semi-robust and accessible way to test this, I don't know how to include and validate this.

Reply:
That's OK with me. Maybe we can find somebody with a GPU who is willing to give it a try.